### PR TITLE
Added Rooboard 65

### PR DESCRIPTION
**Pull request by mrnoisytiger**
## Description

Added Rooboard65

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/skippys_custom_pcs/rooboard65/config.h [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x36b6 // Skippys Custom PCs
```
Comment (drashna) [inline keyboards/skippys_custom_pcs/rooboard65/config.h:23]:
```suggestion
#define VENDOR_ID    0x36B6 // Skippys Custom PCs
```

Also, this vender ID is very close to the currently issues VIDs, and makes me uncomfortable.

Comment (mrnoisytiger) [reply keyboards/skippys_custom_pcs/rooboard65/config.h:23]: > this vender ID is very close to the currently issues VIDs, and makes me uncomfortable.
This is merely what my client requested that I use. I have no say in the matter. If it were me, I would've put it under my own VID. Alas.

**Review by drashna — APPROVED** [review]

File: keyboards/skippys_custom_pcs/rooboard65/readme.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+* Per-Key RGB LED's
+
+---
+
+* Keyboard Maintainer: FJLabs
+* Hardware Supported: Rooboard65
+
+Make example for this keyboard (after setting up your build environment):
+
+    make fjlabs/rooboard65:default
```
Comment (zvecr) [inline keyboards/skippys_custom_pcs/rooboard65/readme.md:17]:
```suggestion
    make skippys_custom_pcs/rooboard65:default
```

File: keyboards/skippys_custom_pcs/rooboard65/config.h [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *         ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+*/
+
+// Checked with Eagle Schematic
```
Comment (zvecr) [inline keyboards/skippys_custom_pcs/rooboard65/config.h:44]:
```suggestion
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/skippys_custom_pcs/rooboard65/rules.mk [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
```
Comment (fauxpark) [inline keyboards/skippys_custom_pcs/rooboard65/rules.mk:19]:
```suggestion
```

File: keyboards/skippys_custom_pcs/rooboard65/rules.mk [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+
```
Comment (fauxpark) [inline keyboards/skippys_custom_pcs/rooboard65/rules.mk:25]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/skippys_custom_pcs/rooboard65/rules.mk [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
```
Comment (noroadsleft) [inline keyboards/skippys_custom_pcs/rooboard65/rules.mk:19]:
```suggestion
NKRO_ENABLE = yes           # Enable N-Key Rollover
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]
Just this one thing.

File: keyboards/skippys_custom_pcs/rooboard65/config.h [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+
+/* disable print */
+//#define NO_PRINT
+
+/* disable action features */
+//#define NO_ACTION_LAYER
+//#define NO_ACTION_TAPPING
+//#define NO_ACTION_ONESHOT
+#define NO_ACTION_MACRO
+#define NO_ACTION_FUNCTION
```
Comment (drashna) [inline keyboards/skippys_custom_pcs/rooboard65/config.h:119]:
```suggetion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/skippys_custom_pcs/rooboard65/config.h (hunk 7 above)
Comment (mrnoisytiger) [reply keyboards/skippys_custom_pcs/rooboard65/config.h:119]: Weirdly, this is not giving me the option to accept the suggestion...
Screen Shot 2022-04-18 at 7 52 43 PM

Comment (drashna) [reply keyboards/skippys_custom_pcs/rooboard65/config.h:119]: Whoops, forgot a letter. 

But yeah, these defines are no longer needed, as action macros an action functions have been completely removed.

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**